content/docs/user-guide/project-config/project-manager.md

```diff
@@ -0,0 +1,151 @@
+{{< /note >}}
+
+![Engine tab with default values](/images/user-guide/project-config/project-manager/engine-tab.png)
+
+The Engine tab contains the following O3DE settings:
+
+| Setting | Description |
+| - | - |
+| **Engine Version** | (Read-only) Displays the O3DE engine version read from the `engine.json` manifest of the engine associated with Project Manager when Project Manager was launched. |
+| **3rd Party Software Folder** | Defines the location of the downloadable packages used by O3DE and its components. |
+| **Default Projects Folder** | Defines the default folder for projects. New projects will be created in this folder unless a different path is specified during the new project workflow. |
+| **Default Gems Folder** | Defines the default folder for Gems. New Gems will be created in this folder unless a different path is specified when the Gem is created. |
+| **Default Project Templates Folder** | Defines the default folder for project templates. New project templates will be created in this folder unless a different path is specified when the project template is created. |
+
+### Project details
+
+The **Enter Project Details** screen is part of the new project workflow. On this screen you can set the project's name and location on your computer. You can also choose the project template, which defines the initial set of Gems that are enabled in your new project. You can further refine the initial set of Gems using the **Configure Gems** button.
+
+![Create a New Project - Project Details screen](/images/welcome-guide/project-manager-create-project.png)
+
+### Project settings
+
+The **Edit Project Settings** menu action in the project context menu opens the **Edit Project Settings** screen. On this screen you can change the project's display name and update the project's icon. These settings and more are stored in the `project.json` file located at the root of your project directory. You can also change the set of Gems that are enabled for your project using the **Configure Gems** button.
+
+![Edit Project Settings screen](/images/user-guide/project-config/project-manager/project-settings.png)
+
+The Edit Project Settings screen contains the following project settings:
+
+| Setting | Description |
+| - | - |
+| **Project Name** | The project's display name. This is the name that's displayed under the project icon on the Projects tab in Project Manager. |
+| **Project Location** | The project's location on your computer. If you select a different folder using the folder button, Project Manager will move your project to the new location and update the O3DE manifest with the new path. A project rebuild is required after moving your project in this way. |
+| **Project Preview** | The path to the project icon image. The filename will always be `preview.png`. If you select a different image using the folder button, Project Manager will copy the image to your project folder and rename it `preview.png`. |
```

Comment:
```suggestion
| **Project Preview** | The path to the project icon image. The file name will always be `preview.png.` If you select a different image using the folder button, Project Manager will copy the image to your project folder and rename it `preview.png.` |
```

Reply:
I'm going to reject this suggestion because the back quotes delineate a formatting region (fixed width font) and I don't want the period to be included in this--it would appear as though it were part of the filename. (Also, I'm going to keep filename as one word, since both versions appear to be widely accepted.)